src/utils/framework-server.ts

```diff
@@ -44,7 +44,9 @@ export const startFrameworkServer = async function ({
     await rm(settings.dist, { recursive: true, force: true })
   }
 
-  runCommand(settings.command, { env: settings.env, spinner, cwd })
+  if (settings.command) {
+    runCommand(settings.command, { env: settings.env, spinner, cwd })
+  }
```

Comment:
If you scroll up you'll see we were already doing this in one branch but not the other. Adding some type safety uncovered this.